production/helm/loki/values.yaml

```diff
@@ -1059,7 +1059,7 @@ tracing:
 # Configuration for `minio` child chart
 # -------------------------------------
 minio:
-  enabled: true
+  enabled: false 
```

Comment:
this looks like a bug, do we have the `minio` section twice? I think we need to merge this `enabled` into the section below it

Reply:
wait, nevermind, the diff was doing something weird for me, this is good